lib/machinegun.rb

```diff
@@ -3,10 +3,10 @@
 
 class MachineGun
   
-  def self.run
+  def self.run interval = 0.5
```

Comment:
I forgot that the nil ENV var will override the default interval value. How about this?

```ruby
def self.run opts = {}
  interval = opts[:interval] || 0.5
```

(`||` is fine here. I can't remember how we handled `and` and `or` on assignment.)